frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx

```diff
@@ -65,12 +65,23 @@ const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
   const [requestedUnit, setRequestedUnit] = React.useState(defaultSize[1] || 'Ti');
   const [validSize, setValidSize] = React.useState(true);
   const pvcAccessMode = getPVCAccessModes(resource, 'title');
+  const [pvcSC, setPVCStorageClass] = React.useState('');
```

Comment:
if we initialise this with getName(scResource) then this or logic is not required right? https://github.com/openshift/console/pull/13549/files#diff-bb0f2a90c2221d69e37016006299c41456aca7ccb7ed8e367d7674b0207757adR71

Reply:
correct me if I am wrong here

Reply:
I am following this from the restore model properties and its working good. Also observed scResource taking some time to load.

Reply:
alright then leave it